src/constants/notification-content.js

```diff
@@ -0,0 +1,167 @@
+export default function getNotificationContent({ type, payload }) {
+  let fragments;
+  switch (type) {
+    case "purchase_status_changed": {
+      fragments = [
+        'Your',
+        {
+          text: (payload.product.type == null ?
+            payload.product.name : `'${payload.product.name}' ${payload.product.type}`),
+          url: '/products/' + payload.product.id,
+        },
+        'purchase',
+        (payload.stock_change.status === 'ready_for_pickup' ?
+          'is ready to be picked up at 319 Office' : 'was rejected'),
```

Comment:
Spaces missing everywhere. After "your", around "purchase", ...
and in all other notification types

Reply:
Yeah, this is something I hoped HTML would take care of, but it didn't, so we will have to resort to something else, haven't yet come up with a remedy

Reply:
I will add the spaces, however, we'll see if that works